core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -244,7 +244,9 @@ Slave.Launching={0} Launching slave agent
 Slave.Network.Mounted.File.System.Warning=Are you sure you want to use network mounted file system for FS root? Note that this directory does not need to be visible to the master.
 Slave.Remote.Director.Mandatory=Remote directory is mandatory
 Slave.Terminated={0} slave agent was terminated
-Slave.the_remote_root_must_be_an_absolute_path=The remote root must be an absolute path.
+Slave.Remote.Relative.Path.Warning=Are you sure you want to use a relative path for the FS root? Note that relative \
+  paths require that you can assure that the selected launcher provides a consistent current working directory. Using \
```

Comment:
I think Stephen is trying to say something about cloudy slaves that share a common filesystem despite apparently different launch details, but I am not really sure either. Deserves a clearer explanation.